blueoil/cmd/init.py

```diff
@@ -131,7 +129,9 @@
     ("cosine", "'cosine' -> cosine learning rate."),
     ("2-step-decay", "'2-step-decay' -> learning rate decrease by 1/10 on {epochs}/2 and {epochs}-1."),
     ("3-step-decay", "'3-step-decay' -> learning rate decrease by 1/10 on {epochs}/3 and {epochs}*2/3 and {epochs}-1"),
-    ("3-step-decay-with-warmup", "'3-step-decay-with-warmup' -> warmup learning rate 1/1000 in first epoch, then train the same way as '3-step-decay'"),
+    ("3-step-decay-with-warmup",
```

Comment:
It is recommended to write long sentences like this!

```python
s = (
    "spam "
    "egg "
    "spam"
)

print(s == "spam egg spam")
```

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
@odoku changed. this is what you want?